pkg/scheduler/testing/wrappers.go

```diff
@@ -608,14 +632,36 @@ func (p *PodWrapper) Annotations(annotations map[string]string) *PodWrapper {
 	return p
 }
 
+// Res adds a new container to the inner pod with given resource map.
+func (p *PodWrapper) Res(resMap map[v1.ResourceName]string) *PodWrapper {
+	if len(resMap) == 0 {
+		return p
+	}
+
+	name := fmt.Sprintf("con%d", len(p.Spec.Containers))
+	p.Spec.Containers = append(p.Spec.Containers, MakeContainer().Name(name).Image(imageutils.GetPauseImageName()).Resources(resMap).Obj())
+	return p
+}
+
 // Req adds a new container to the inner pod with given resource map.
 func (p *PodWrapper) Req(resMap map[v1.ResourceName]string) *PodWrapper {
 	if len(resMap) == 0 {
 		return p
 	}
 
 	name := fmt.Sprintf("con%d", len(p.Spec.Containers))
-	p.Spec.Containers = append(p.Spec.Containers, MakeContainer().Name(name).Image(imageutils.GetPauseImageName()).Resources(resMap).Obj())
+	p.Spec.Containers = append(p.Spec.Containers, MakeContainer().Name(name).Image(imageutils.GetPauseImageName()).ResourceRequests(resMap).Obj())
```

Comment:
Rename resMap to reqMap, short for requestsMap.

Reply:
Done